app/views.py

```diff
@@ -0,0 +1,165 @@
+from flask import Flask, request
+from flask_restful import Resource, Api
+from app.models import User, Qn, Answer
+
+
+app = Flask(__name__)
+api = Api(app, prefix="/api/v1")
+
+
+def get_qn_by_id(qn_id):
+    for qn in Qn.questions:
+        if qn.get('qn_id') == int(qn_id):
+            return qn
+
+
+class SubscriberCollection(Resource):
+
```

Comment:
Right before each function to be documented, add a swag_from decorator to read the respective docs in the yaml file

Reply:
thanx much for this clue